tuning/src/main/java/frc/robot/Robot.java

```diff
@@ -80,6 +84,7 @@ public void teleopPeriodic() {
     elevator.updatePIDs();
 
     elevator.teleopPeriodic();
+    algaePivot.teleopPeriodic();
```

Comment:
Comment this out for now, typically we only want to tune one subsystem at a time